builds/mariner/SPECS/aziot-edge/aziot-edge.spec

```diff
@@ -0,0 +1,189 @@
+%define iotedge_user iotedge
+%define iotedge_group %{iotedge_user}
+%define iotedge_home %{_localstatedir}/lib/aziot/edged
+%define iotedge_logdir %{_localstatedir}/log/aziot/edged
+%define iotedge_socketdir %{_localstatedir}/lib/iotedge
+%define aziot_confdir %{_sysconfdir}/aziot
+%define iotedge_confdir %{aziot_confdir}/edged
+
+Name:           aziot-edge
+Version:        @@VERSION@@
+Release:        3%{?dist}
+
+License:        MIT
+Summary:        Azure IoT Edge Module Runtime
+URL:            https://github.com/azure/iotedge
+
+%{?systemd_requires}
+BuildRequires:  systemd
+BuildRequires:  rust >= 1.47.0
+Requires(pre):  shadow-utils
+Requires:       openssl
+Requires:       moby-engine
+Requires:       moby-cli
```

Comment:
Why do we require `openssl`, `moby-engine`, `moby-cli`? 
CentOs doesn't seem to need them. Could you give remove these and give it a shot pls?